src/Sulu/Bundle/TagBundle/Infrastructure/Sulu/Content/ResourceLoader/TagResourceLoader.php

```diff
@@ -32,11 +32,11 @@ public function __construct(
 
     public function load(array $ids, ?string $locale, array $params = []): array
     {
-        $result = $this->tagRepository->findBy(['id' => $ids]);
+        $result = $this->tagRepository->findBy(['name' => $ids]);
```

Comment:
No I it doesn't

But tbh I would like it more to save the `id` instead of the `name`, but then we would need to adjust it also on the FE 🤔 

What would be the best way to go forward?